tests/functional/fx_fennec_v1_force_auth.js

```diff
@@ -5,31 +5,35 @@
 define([
   'intern!object',
   'tests/lib/helpers',
-  'tests/functional/lib/helpers'
-], function (registerSuite, TestHelpers, FunctionalHelpers) {
-  var thenify = FunctionalHelpers.thenify;
-
-  var clearBrowserState = FunctionalHelpers.clearBrowserState;
-  var closeCurrentWindow = FunctionalHelpers.closeCurrentWindow;
-  var createUser = FunctionalHelpers.createUser;
-  var fillOutForceAuth = FunctionalHelpers.fillOutForceAuth;
-  var fillOutSignInUnblock = FunctionalHelpers.fillOutSignInUnblock;
-  var openForceAuth = FunctionalHelpers.openForceAuth;
-  var openVerificationLinkInDifferentBrowser = FunctionalHelpers.openVerificationLinkInDifferentBrowser;
-  var openVerificationLinkInNewTab = FunctionalHelpers.openVerificationLinkInNewTab;
-  var respondToWebChannelMessage = FunctionalHelpers.respondToWebChannelMessage;
-  var testElementExists = FunctionalHelpers.testElementExists;
-  var testIsBrowserNotified = FunctionalHelpers.testIsBrowserNotified;
-
-  var PASSWORD = 'password';
-  var email;
-
-  var setupTest = thenify(function (options) {
+  'tests/functional/lib/helpers',
+  'tests/functional/lib/selectors'
+], function (registerSuite, TestHelpers, FunctionalHelpers, selectors) {
+  'use strict';
+
+  const {
+    clearBrowserState,
+    closeCurrentWindow,
+    createUser,
+    fillOutForceAuth,
+    fillOutSignInUnblock,
+    openForceAuth,
+    openVerificationLinkInDifferentBrowser,
+    openVerificationLinkInNewTab,
+    respondToWebChannelMessage,
+    testElementExists,
+    testIsBrowserNotified,
+    thenify,
```

Comment:
nit: remove extra `,`

Reply:
I'm going to make a proposal: We keep the trailing `,` where it's allowed by JavaScript, doing so makes it so someone can add lines afterwards without affecting the history of this line.

Reply:
I was going to make counter proposal to add it as a lint rule :) I don't mind either way as long as everything is consistent.

Reply:
> add it as a lint rule

I like this plan.